origin-growth/migrations/20190312002247-update-growth-referral.js

```diff
@@ -0,0 +1,62 @@
+'use strict'
+
+const tableName = 'growth_referral'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.dropTable(tableName)
+      .then(() => 
+        queryInterface.createTable(tableName, {
+          id: {
```

Comment:
can you update origin-growth/src/models/growth_referral.js to add the new id column ?

Reply:
yup